aws_lambda_powertools/utilities/data_classes/api_gateway_proxy_event.py

```diff
@@ -184,6 +194,16 @@ def get_lambda(self) -> Optional[Dict[str, Any]]:
         """Lambda authorization context details"""
         return self.get("lambda")
 
+    def get_context(self) -> Optional[Dict[str, Any]]:
+        """Retrieve the authorization context details for Lambda.
+
+        Returns:
+        --------
+            Optional[Dict[str, Any]]
+            A dictionary containing Lambda authorization context details, or None if the information is not available.
+        """
+        return self.get("lambda")
```

Comment:
```suggestion
        return self.get("lambda", {})
```

that way customers don't have to handle themselves with an `or`.